contributors/devel/how-to-doc.md

```diff
@@ -117,7 +117,6 @@ spec:
     - containerPort: 80
 ```
 
-[Download example](../../test/fixtures/doc-yaml/user-guide/pod.yaml?raw=true)
```

Comment:
This shouldn't be removed. The correct link is: https://git.k8s.io/kubernetes/test/fixtures/doc-yaml/user-guide/pod.yaml?raw=true

Reply:
@cblecker which one is preferred, `git.k8s.io` or `github.com/kubernetes`?

Reply:
`git.k8s.io` where we can. It's an abstraction we control. But not a big deal if we use github.

Reply:
Thanks!